README.md

```diff
@@ -1,146 +1,171 @@
 <a name="readme-top"></a>
 
+
+
+
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Usage](#usage)  
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-- 🔭 [Future Feature](#futurefeatures)
-- 🤝 [Contributing](#contributing)
-- ⭐️ [Show your support](#showyoursupport)
-- 🙏 [Acknowledgements](#acknowledgements)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [portofolio] <a name="about-project"></a>
+# 📖 Felipe Portfolio <a name="about-project"></a>
 
-> This project is the biginin of my mobile portfolio.
+> Describe your project in 1 or 2 sentences.
 
-**[portofolio]** is my personal portofolio.
+**Felipe Portfolio** is a responsive webpage who can open in mobile and desktop too.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Tech stacks used on the project are.
+HTML5  CSS3 JS
+
 
-<details>
-  
-  <ul>
-    <li><a href="#">html</a></li>
-    <li><a href="#">css</a></li>
-  </ul>
-</details>
+<!-- Features -->
 
-<details>
-  <summary>Mobile portfolio middle section: About and work</summary>
-  <ul>
-    
-  </ul>
-</details>
+### Key Features <a name="key-features"></a>
 
 
+- **LOVE**
+- **PASSION**
+- **CODING**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://bleu-yves-sopoude.github.io/#)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
+
 To get a local copy up and running, follow these steps.
+Create a GitHub acount an clone this repository
 
 ### Prerequisites
 
 In order to run this project you need:
 
+a Browser who support Html5 and Css3
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
 
 ### Setup
 
 Clone this repository to your desired folder:
 
 
-### Install
+  cd my-folder
+  git clone https://github.com/Bleu-Yves-Sopoude/mb-portfolio.git
 
-Install this project with:
-npm i
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Usage
+<!-- AUTHORS -->
 
-To run the project, open index.html
+## 👥 Authors <a name="authors"></a>
 
+> Mention all of the collaborators of this project.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+👤 **Bleu Yves Sopoude**
```

Comment:
[Optional]

- [ ]  Kindly include your  social media handle details  linkedin link and your twiter link this will enable a recruiter to cantact you easily in future